src/uct/ib/base/ib_md.c

```diff
@@ -309,7 +309,7 @@ void *uct_ib_md_mem_handle_thread_func(void *arg)
     while (ctx->length > 0) {
         if (ctx->params != NULL) {
             status = uct_ib_reg_mr(ctx->md, ctx->address, length, ctx->params,
-                                   ctx->access_flags, NULL, &ctx->mrs[mr_idx]);
+                                   ctx->access_flags, NULL, &ctx->mrs[mr_idx], 1);
```

Comment:
why hide_errors is true here?

Reply:
@yosefe Oh, I actually wanted to put here a value that doesn't change the previous behavior and put `1` by mistake. I'll also change it in the other places.